modules/administration/pages/public-cloud-azure.adoc

```diff
@@ -0,0 +1,212 @@
+[[public-cloud-azure]]
+= SUSE Manager Server in Azure
+
+
+
+== Resource Requirements
+
+
+
+=== Image and VM (Virtual Machine)
```

Comment:
Ooh, I'm not a fan of the nested headings. Can we either delete some, or add some text in between?